mm-mac2ipv4.bash

```diff
@@ -23,7 +23,7 @@ function list-all {
   for ip2 in `seq 0 255`;do
     mac4=$(printf "%02X\n" $ip2)
 
-    ((ip2=ip2%64 + 64))
+    ((ip2=ip2 % 32 + 96))
```

Comment:
I didn't catch this in the original, but it's almost never a good idea to modify a variable which is the element of a collection under iteration. Many languages allow it, but it's a bad habit that too often leads to bugs.

I'll let it through, but please consider changing it.